### ISSUE TITLE
axis=1 missing in In Code4.36 of notebooks/04_geocentric_models.ipynb

### ISSUE THREAD

**Issue by himalayajung** [fehiepsi/rethinking-numpyro#14]
In Code4.36 of notebooks/04_geocentric_models.ipynb,

mu, sigma = jnp.mean(samples_flat, axis=1), jnp.cov(samples_flat) # axis =1 is missing

**Comment by fehiepsi** [comment]
Thanks, @himalayajung! I just fixed it upstream.
